### PR TITLE
Changing systemd command sets

### LINKED ISSUE #539: global: change systemd service names: ceph-$FSID@$DAEMON_NAME
```
asettle@linux-nrkp:~/susedoc/doc-ses> grep -r "systemctl"
Binary file .git/objects/pack/pack-39c2ee5729a2d39cf868026729be84b25c28ad2d.pack matches
xml/admin_ceph_gateway.xml:<screen>&prompt.cephuser.ogw;systemctl start ceph-radosgw@rgw.`hostname -s`

^^^ outdated. doesn't follow ceph-$FSID@$DAEMON_NAME

xml/admin_ceph_gateway.xml:&prompt.cephuser.ogw;systemctl enable ceph-radosgw@rgw.`hostname -s`</screen>

^^^ outdated. doesn't follow ceph-$FSID@$DAEMON_NAME

xml/admin_ds_backup.xml:&prompt.smaster;<command>systemctl</command> enable salt-master
xml/admin_ds_backup.xml:&prompt.smaster;<command>systemctl</command> start salt-master
xml/admin_ds_backup.xml:&prompt.smaster;<command>systemctl</command> enable salt-minion
xml/admin_ds_backup.xml:&prompt.smaster;<command>systemctl</command> start salt-minion
xml/admin_ds_backup.xml:&prompt.smaster;<command>systemctl</command> restart salt-minion
xml/admin_operating_services.xml:  <command>systemctl</command> command.
xml/admin_operating_services.xml:   By using the <command>systemctl</command> command, you can operate &ceph;
xml/admin_operating_services.xml:&prompt.root;systemctl list-units \-\-all \-\-type=service,target ceph*
xml/admin_operating_services.xml:&prompt.root;systemctl list-units \-\-all \-\-state=inactive \-\-type=service ceph*
xml/admin_operating_services.xml: "systemctl list-units \-\-all \-\-type=service ceph* | sed -e '/^$/,$ d'"
xml/admin_operating_services.xml:<screen>&prompt.sminion;systemctl stop ceph.target</screen>
xml/admin_operating_services.xml:<screen>&prompt.sminion;systemctl restart ceph-b4b30c6e-9681-11ea-ac39-525400d7702d.target</screen>
xml/admin_operating_services.xml:<screen>systemctl <replaceable>COMMAND</replaceable> <replaceable>SERVICE_NAME</replaceable></screen>
xml/admin_operating_services.xml:<screen>&prompt.root;systemctl restart ceph-b4b30c6e-9681-11ea-ac39-525400d7702d@osd.1.service</screen>
xml/admin_operating_services.xml:<screen>&prompt.root;systemctl status ceph-b4b30c6e-9681-11ea-ac39-525400d7702d@osd.0.service</screen>
xml/admin_operating_services.xml:<screen>&prompt.smaster;salt <replaceable>TARGET</replaceable> cmd.run "systemctl stop ceph.target"</screen>
xml/bp_troubleshooting_cephadm.xml: <sect1 xml:id="bp-troubleshooting-cephadm-systemctl">
xml/bp_troubleshooting_cephadm.xml:systemctl status "ceph-$(cephadm shell ceph fsid)@<replaceable>service name</replaceable>.service";
xml/bp_troubleshooting_cephadm.xml:    systemctl status "ceph-$fsid@$name.service" > $name;
xml/bp_troubleshooting_objectgateway.xml:&prompt.cephuser;sudo systemctl restart ceph-radosgw.service

^^^ outdated. doesn't follow ceph-$FSID@$DAEMON_NAME


xml/bp_troubleshooting_osds.xml:<screen>&prompt.root;systemctl start ceph-osd@2.service</screen>

^^^ outdated. doesn't follow ceph-$FSID@$DAEMON_NAME


xml/leftovers.xml:<screen>&prompt.cephuser.ogw;systemctl restart ceph-radosgw@rgw.`hostname -s`</screen>

^^^ outdated. doesn't follow ceph-$FSID@$DAEMON_NAME


xml/admin_ceph_iscsi.xml:      On &sls; (SLES); and &rhel; (RHEL) hosts, run <command>systemctl start
xml/admin_ceph_iscsi.xml:      <command>systemctl</command> is not available).
xml/admin_ceph_iscsi.xml:      On Debian and Ubuntu hosts, run <command>systemctl start
xml/admin_ceph_iscsi.xml:<screen>&prompt.root;systemctl restart multipathd</screen>
xml/admin_operating_monitor.xml:&prompt.cephuser.osd;sudo systemctl start ceph-<replaceable>CLUSTER_ID</replaceable>@osd.0.service
xml/admin_rbd.xml:<screen>&prompt.sminion;systemctl enable ceph-rbd-mirror@rbd-mirror.<replaceable>UNIQUE-ID</replaceable></screen>

^^^ outdated. doesn't follow ceph-$FSID@$DAEMON_NAME

xml/deployment_cifs.xml:&prompt.root;systemctl restart smb.service
xml/deployment_cifs.xml:&prompt.root;systemctl restart nmb.service
xml/deployment_cifs.xml:&prompt.root;systemctl restart winbind.service
xml/deployment_cifs.xml:&prompt.root;systemctl enable smb.service
xml/deployment_cifs.xml:&prompt.root;systemctl enable nmb.service
xml/deployment_cifs.xml:&prompt.root;systemctl enable winbind.service
xml/deployment_cifs.xml:<screen>&prompt.root;systemctl disable ctdb
xml/deployment_cifs.xml:&prompt.root;systemctl disable smb
xml/deployment_cifs.xml:&prompt.root;systemctl disable nmb
xml/deployment_cifs.xml:&prompt.root;systemctl disable winbind
xml/deployment_cifs.xml:&prompt.root;systemctl stop ctdb
xml/deployment_cifs.xml:&prompt.root;systemctl stop smb
xml/deployment_cifs.xml:&prompt.root;systemctl stop nmb
xml/deployment_cifs.xml:&prompt.root;systemctl stop winbind</screen>
xml/deployment_cifs.xml:&prompt.earth;systemctl enable rbdmap.service &amp;&amp; systemctl start rbdmap.service
xml/deployment_cifs.xml:&prompt.mars;systemctl enable rbdmap.service &amp;&amp; systemctl start rbdmap.service
xml/deploy_cephadm.xml:<screen>&prompt.smaster;systemctl enable salt-master.service
xml/deploy_cephadm.xml:&prompt.smaster;systemctl start salt-master.service</screen>
xml/deploy_cephadm.xml:<screen>&prompt.sminion;systemctl restart salt-minion.service</screen>
xml/deploy_cephadm.xml:<screen>&prompt.root;systemctl enable salt-minion.service
xml/deploy_cephadm.xml:&prompt.root;systemctl start salt-minion.service</screen>
xml/deploy_cephadm.xml:&prompt.smaster;systemctl restart salt-master.service
xml/deployment_hwrecommend.xml:     will not start correctly (running <command>systemctl status
xml/deployment_hwrecommend.xml:<screen>&prompt.root;systemctl restart ceph-*@osd.*.service</screen>
xml/admin_ceph_upgrade.xml:<screen>&prompt.sminion;systemctl disable ntpd.service &amp;&amp; systemctl stop ntpd.service</screen>
xml/admin_ceph_upgrade.xml:<screen>&prompt.sminion;systemctl start chronyd.service &amp;&amp; systemctl enable chronyd.service</screen>
xml/admin_ceph_upgrade.xml:<screen>&prompt.sminion;systemctl disable ntpd.service &amp;&amp; systemctl stop ntpd.service</screen>
xml/admin_ceph_upgrade.xml:<screen>&prompt.sminion;systemctl start chronyd.service &amp;&amp; systemctl enable chronyd.service</screen>
xml/admin_ceph_upgrade.xml:Running command: /bin/systemctl start ceph-osd@8
xml/admin_ceph_upgrade.xml: salt $OSDNODE* cmd.run "systemctl stop ceph-osd@$OSD";\
xml/admin_ceph_upgrade.xml: salt $OSDNODE* cmd.run "systemctl start ceph-osd@$OSD";\
xml/admin_saltcluster.xml:&prompt.smaster;systemctl stop salt-master.service
xml/admin_saltcluster.xml:&prompt.smaster;systemctl stop salt-minion.service
xml/admin_saltcluster.xml:&prompt.sminion;systemctl restart salt-minion.service
xml/bp_troubleshooting_status.xml:<screen>&prompt.root;systemctl stop ceph-osd@123

^^^ outdated. doesn't follow ceph-$FSID@$DAEMON_NAME


xml/bp_troubleshooting_status.xml:&prompt.root;systemctl start ceph-osd@123</screen>

^^^ outdated. doesn't follow ceph-$FSID@$DAEMON_NAME


xml/bp_troubleshooting_tips.xml:<screen>systemctl stop <replaceable>OSD_SERVICE_NAME</replaceable></screen>

^^^ outdated. doesn't follow ceph-$FSID@$DAEMON_NAME

xml/bp_troubleshooting_tips.xml:  &prompt.sminion;systemctl start ceph-<replaceable>FSID</replaceable>@osd.<replaceable>OSD_NUMBER</replaceable>.service
xml/bp_troubleshooting_tips.xml:<screen>&prompt.root;systemctl stop ceph.target</screen>
xml/bp_troubleshooting_tips.xml:<screen>&prompt.root;systemctl status chronyd.service</screen>
xml/bp_troubleshooting_tips.xml:<screen>&prompt.root;systemctl start <replaceable>target</replaceable></screen>
xml/bp_troubleshooting_tips.xml:<screen>&prompt.smaster;systemctl stop salt-master</screen>
xml/bp_troubleshooting_tips.xml:<screen>&prompt.smaster;systemctl start salt-master</screen>

```

### PR DESCRIPTION
**Pull request by asettle**
Closes #539

### CODE REVIEW

File: xml/bp_troubleshooting_tips.xml [hunk 1]
```diff
@@ -1031,7 +1031,7 @@ ID WEIGHT TYPE NAME    UP/DOWN REWEIGHT PRIMARY-AFFINITY
      <para>
       Stop the &smaster; service:
      </para>
-<screen>&prompt.smaster;systemctl stop salt-master</screen>
```
Comment (sebastian-philipp) [inline xml/bp_troubleshooting_tips.xml:1034]: nope. salt-master is still salt-master

File: xml/admin_ceph_gateway.xml [hunk 2]
```diff
@@ -2879,8 +2879,14 @@ rgw_override_bucket_index_max_shards = 12
     <para>
      On the &ogw; host, start and enable the &cogw; service:
     </para>
-<screen>&prompt.cephuser.ogw;systemctl start ceph-radosgw@rgw.`hostname -s`
-&prompt.cephuser.ogw;systemctl enable ceph-radosgw@rgw.`hostname -s`</screen>
+<screen>&prompt.cephuser.ogw;systemctl start ceph-<replaceable>FSID</replaceable>@<replaceable>DAEMON_NAME</replaceable>
+&prompt.cephuser.ogw;systemctl enable ceph-<replaceable>FSID</replaceable>@<replaceable>DAEMON_NAME</replaceable></screen>
+   <note>
+     <para>
+       To ensure you have the relevant daemon name, run the following command:
```
Comment (asettle) [inline xml/admin_ceph_gateway.xml:2886]: @tbazant do you think we should include this note everywhere?

Comment (tbazant) [reply xml/admin_ceph_gateway.xml:2886]: @asettle I would make this note a tip and prepend it before the `systemctl` commands, saying something like "To identify the unique FSID of the cluster, run `ceph fsid`. To identify the &ogw; daemon name, run `ceph orch ps ---hostname &lt;replaceable>HOSTNAME&lt;/replaceable>`"

File: xml/admin_ceph_gateway.xml [hunk 3]
```diff
@@ -2879,8 +2879,14 @@ rgw_override_bucket_index_max_shards = 12
...
+     </para>
+<screen>&prompt.cephuser.ogw;ceph orch ps ---hostname $(hostname)</screen>
```
Comment (tbazant) [inline xml/admin_ceph_gateway.xml:2888]:
```suggestion
<screen>&prompt.cephuser.ogw;ceph orch ps ---hostname <replaceable>HOSTNAME</replaceable></screen>
```

File: xml/admin_ceph_gateway.xml (hunk 2 above)
Comment (tbazant) [reply xml/admin_ceph_gateway.xml:2886]: my second thought is that we have a chapter about operating Ceph services, so....i would just refer it not to cause too much noise and mess
see https://susedoc.github.io/doc-ses/master/single-html/ses-admin/#ceph-operating-services-finding-names

File: xml/bp_troubleshooting_status.xml [hunk 4]
```diff
@@ -433,7 +433,7 @@
             The old OSDs can be updated to use the new usage tracking scheme by
             stopping each OSD, running a repair operation, and the restarting it.
             For example, if <literal>osd.123</literal> requires an update:</para>
-<screen>&prompt.root;systemctl stop ceph-osd@123
+<screen>&prompt.root;systemctl stop ceph-<replaceable>FSID</replaceable>@<replaceable>DAEMON_NAME</replaceable>
```
Comment (tbazant) [inline xml/bp_troubleshooting_status.xml:436]: here it's a specific daemon named `osd.123`:
```suggestion
<screen>&prompt.root;systemctl stop ceph-<replaceable>FSID</replaceable>@<replaceable>osd.123</replaceable>
```

File: xml/bp_troubleshooting_osds.xml [hunk 5]
```diff
@@ -769,7 +769,7 @@ osd.0 is down since epoch 23, last address 192.168.106.220:6800/11080
    <filename>/var/log/ceph/ceph-osd.2.log</filename>. After you find and fix
    the reason why the OSD is not running, start it with
   </para>
-<screen>&prompt.root;systemctl start ceph-osd@2.service</screen>
+<screen>&prompt.root;systemctl start ceph-<replaceable>FSID</replaceable>@<replaceable>DAEMON_NAME</replaceable></screen>
```
Comment (tbazant) [inline xml/bp_troubleshooting_osds.xml:772]:
```suggestion
<screen>&prompt.root;systemctl start ceph-<replaceable>FSID</replaceable>@<replaceable>osd.2</replaceable></screen>
```

**Review by tbazant — CHANGES_REQUESTED** [review]
Left a few comments

File: xml/admin_ceph_gateway.xml (hunk 2 above)
Comment (asettle) [reply xml/admin_ceph_gateway.xml:2886]: @sebastian-philipp either of those options suit you?

Comment (sebastian-philipp) [reply xml/admin_ceph_gateway.xml:2886]: I honestly don't mind. whatever you thing is better.

**Review by tbazant — APPROVED** [review]
lgtm

**Pull request merged**